VRCFaceTracking/Assets/UI/App.xaml.cs

```diff
@@ -10,8 +12,37 @@ public partial class App
         
         public App()
         {
-            BindingOperations.EnableCollectionSynchronization(Logger.ConsoleOutput, Logger.ConsoleLock);
-            _oscThread.Start();
+            try
+            {
+                BindingOperations.EnableCollectionSynchronization(Logger.ConsoleOutput, Logger.ConsoleLock);
+                _oscThread.Start();
+            }
+            catch ( Exception e ) 
+            {
+                string filePath = Utils.PersistentDataDirectory + "/ExceptionLog.txt";
```

Comment:
We should try and move this to the logger in the future, but fine for now as the file logging functionality is yet to be fully implemented. Just something to keep in mind 👍

Reply:
Sounds good, I believe I did this really early on in the PR and neglected to finalize it further.